contracts/libraries/OperatorLib.sol

```diff
@@ -43,8 +44,7 @@ library OperatorLib {
                 if (!increaseValidatorCount) {
                     operator.validatorCount -= deltaValidatorCount;
                 } else if (
-                    (operator.validatorCount += deltaValidatorCount) >
-                    s.validatorsPerOperatorLimit
+                    (operator.validatorCount += deltaValidatorCount) > SSVStorageProtocol.load().validatorsPerOperatorLimit
```

Comment:
Its loaded 4 times, isnt better to load it once?

Reply:
loaded 4 times: 116170 gas
cached into memory variable: 116412
@andrew-blox